code/client/src/sdk/mafia/ui/hud/c_hud_controller.h

```diff
@@ -1,9 +1,44 @@
 #pragma once
 
+#include <utils/hooking/hooking.h>
+
 namespace SDK {
     namespace mafia::ui::hud {
+
+        class C_RaceTimer {
+          public:
+
+            // NB: Do not use in MP!
+            // This starts C_RaceManager!
+            // Use mafia::ui::hud::RaceXBin instead!
+            void SetVisible(const bool visible) {
+                hook::this_call(0x14304F520, this, visible);
+            }
+
+            // NB: Do not use in MP!
+            // This starts C_RaceManager!
+            // Use mafia::ui::hud::RaceXBin instead!
+            void StartRace(const uint32_t NumCheckpoints, const float TargetTime, const uint32_t NumLaps) {
```

Comment:
member naming is wrong, should start lowercase

Reply:
member names are now lowercase